### ISSUE TITLE
Ability to accept decompilation legal agreement globally for all current and future workspaces.

### ISSUE THREAD

**Issue by joelday** [OmniSharp/omnisharp-vscode#3982]
## Issue Description ##
As a developer, I would like to be presented with and accept the legal agreement displayed by the `csharp.showDecompilationTerms` command only once instead of when the language support is activated for the first time in each workspace.

Thank you!

**Comment by cartermp** [comment]
Yeah, this would definitely be nice to have. I think there's some conversations going on about the same topic for Visual Studio too, since as you could imagine, lots of users there are fine accepting the terms but don't want to keep seeing it for every VS session.

**Comment by joelday** [comment]
@cartermp Quick note that this is now displayed every time I open vscode. Unsure if others are experiencing that.

**Comment by heshuimu** [comment]
> @cartermp Quick note that this is now displayed every time I open vscode. Unsure if others are experiencing that.

I can confirm this, even when opening workspace that is not using VSCode. It's getting super annoying now.

**Comment by manojbaishya** [comment]
Please vote this issue up:
[Accepting the decompiler agreement permanently](https://github.com/OmniSharp/omnisharp-vscode/issues/4296) 
Since it has more upvotes, maybe product owner will take notice.

**Comment by joelday** [comment]
This shows up constantly even for non C# workspaces now, at least in my case, so it has gone from annoyance to outright bug.

**Comment by joelday** [comment]
@cartermp Should I open a bug to address the dialog appearing for non-C# workspaces?

**Comment by JoeRobich** [comment]
> Should I open a bug to address the dialog appearing for non-C# workspaces?

What type of files are in the workspace? Unfortunately with VS Code it isn't as simple as saying this csproj or sln was opened, so this is a C# workspace. It becomes more a this folder was opened and contains files with these extensions meaning this may be a C# workspace.

**Comment by SeeminglyScience** [comment]
It does the same for me, seems like any time VSCode changes workspaces.  Including just opening a new window with <kbd>Ctrl</kbd> + <kbd>Shift</kbd> + <kbd>N</kbd>.  Here is it popping up for a workspace with just three log files:

![image](https://user-images.githubusercontent.com/24977523/132920038-871d0c69-dd5d-4541-b059-509ddfa56964.png)
